connector/MongoDBAtlasODBC.query.pq

```diff
@@ -1,5 +1,201 @@
-// Use this file to write queries to test your data connector
-let
-    result = MongoDBAtlasODBC.Contents("", "")
-in
-    result
+section MongoDBAtlasODBCUnitTests;
+shared UnitTests = 
+[
+    facts = 
+    {
+        Fact("GetConnectionString - Sets URI option",

```

Comment:
Do you want to add a test where the DB is missing so that you can check that it fails with "Database value cannot be empty"?

Reply:
Ah yes I had a test that did this previously, and removed it when I changed to using the URI option.  The empty database test is there now 👍